fields: 
  'besen':
    start_angle: 0.5
    stop_angle: 12.0
